.githooks/pre-commit.d/20-go-version

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+WORKFLOWS=".github/workflows/verify.yml"
+
+gomod=$(git diff --cached go.mod | grep '^+go ' | sed 's/^.* //g')
```

Comment:
Would it be good to catch the problem even if not part of this commit, i.e. just grep `go.mod` directly without doing git diff(?)

Reply:
Hmm... maybe. Should we then also always automagically enable our git hooks in the cloned repos (by making `install-git-hooks` a prerequisite of the `all` or `build` Makefile target). Otherwise the whole test is only run, and such mistakes are caught, by those (yours, truly) who enable the git hooks by running `make install-git-hooks`... Then again, if we enable the hooks for everyone, how those versions could go out of sync ever again, other people brutally bypassing all hooks using `git commit -n` ?